src/styles/index.css

```diff
@@ -0,0 +1,312 @@
+  content: '11';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(12)::before {
+  content: '12';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(13)::before {
+  content: '13';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(14)::before {
+  content: '14';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(15)::before {
+  content: '15';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(16)::before {
+  content: '16';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(17)::before {
+  content: '17';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(18)::before {
+  content: '18';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(19)::before {
+  content: '19';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(20)::before {
+  content: '20';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(21)::before {
+  content: '21';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(22)::before {
+  content: '22';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(23)::before {
+  content: '23';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(24)::before {
+  content: '24';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(25)::before {
+  content: '25';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(26)::before {
+  content: '26';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(27)::before {
+  content: '27';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(28)::before {
+  content: '28';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(29)::before {
+  content: '29';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(30)::before {
+  content: '30';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar__day:nth-child(31)::before {
+  content: '31';
+  font:
+    30px Arial,
+    sans-serif;
+}
+
+.calendar--month-length-28 :nth-child(-n + 28) {
+  display: flex;
+}
+
+.calendar--month-length-29 :nth-child(-n + 29) {
+  display: flex;
+}
+
+.calendar--month-length-30 :nth-child(-n + 30) {
+  display: flex;
+}
+
+.calendar--month-length-31 :nth-child(-n + 31) {
+  display: flex;
+}
+
+.calendar--start-day-tue .calendar__day:first-child {
+  margin-left: 101px;
+}
+
+.calendar--start-day-wed .calendar__day:first-child {
+  margin-left: 202px;
+}
+
+.calendar--start-day-thu .calendar__day:first-child {
+  margin-left: 303px;
+}
+
+.calendar--start-day-fri .calendar__day:first-child {
+  margin-left: 404px;
+}
+
+.calendar--start-day-sat .calendar__day:first-child {
+  margin-left: 505px;
+}
+
+.calendar--start-day-sun .calendar__day:first-child {
+  margin-left: 606px;
+}
```

Comment:
The '.calendar--start-day-*' classes adjust the margin-left for the first calendar day. Verify that these classes are applied correctly to align the first day of the month according to the starting day of the week.